root/static/scripts/common/components/EntityLink.js

```diff
@@ -448,6 +459,15 @@ component EntityLink(
     );
   }
 
+  if (creditedAs !== null) {
+    parts.push(
+      ' ',
+      bracketed(
+        <span>{texp.l('as “{name}”', {name: entityName})}</span>,
```

Comment:
Embarrassing bug here: this should've been `creditedAs` instead of `entityName`.